src/backend/aspen/app/views/sample.py

```diff
@@ -21,16 +21,46 @@
     GisaidAccession,
     GisaidAccessionWorkflow,
     HostFilteredSequencingReadsCollection,
+    Sample,
     SequencingReadsCollection,
     UploadedPathogenGenome,
     WorkflowStatusType,
 )
-from aspen.database.models.sample import Sample
+from aspen.database.models.sample import RegionType
 from aspen.database.models.usergroup import Group, User
 from aspen.error.recoverable import RecoverableError
 
 SAMPLE_KEY = "samples"
 GISIAD_REJECTION_TIME = datetime.timedelta(days=4)
+SAMPLES_POST_REQUIRED_FIELDS = [
+    "private",
+    "private_identifier",
+    "collection_date",
+    "location",
+    # following fields from PathogenGenome
+    "sequence",
+]
+SAMPLES_POST_OPTIONAL_FIELDS = [
+    "original_submission",
+    "public_identifier",
+    "sample_collected_by",
+    "sample_collector_contact_email",
+    "sample_collector_contact_address",
+    "authors",
+    "division",
+    "country",
+    "region",
+    "organism",
+    "host",
+    "purpose_of_sampling",
+    "specimen_processing",
+    "czb_failed_genome_recovery",
+    # following fields from PathogenGenome
+    "sequencing_depth",
+    "num_unambiguous_sites",
```

Comment:
not huge but not sure we'd ever accept these fields from the user since we calculate them from the sequence now

Reply:
right i'll take those out